src/CoreWCF.Http/tests/ServiceContract/ILifeTimeTestService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CoreWCF;
+
+namespace ServiceContract
+{
+    [ServiceContract]
+    public interface ILifeTimeTestService
+    {
+        [OperationContract]
+        void Start();
+
+        [OperationContract(IsOneWay = true)]
+        void OneWay();
```

Comment:
@mconnew , IsOneWay operation could build but client call would fail, it seems it is not fully supported, right?

Reply:
It should be working. If this isn't working, it's a bug. Please open an issue and add a comment pointing to the issue.

Reply:
Issue #197.